SPECS/xapi.spec

```diff
@@ -551,7 +555,7 @@ Coverage files from unit tests
 - CP-29757: Add new VDI_IS_ENCRYPTED exception
 
 * Tue Nov 27 2018 Christian Lindig <christian.lindig@citrix.com> - 1.137.0-1
-- CP-30039: Generate automatically the release and class files for 
+- CP-30039: Generate automatically the release and class files for
```

Comment:
Tune your editor to not remove trailing spaces on text that we're not the upstream for :)